iamlistening/handler/client.py

```diff
@@ -84,7 +84,7 @@ async def get_latest_message(self):
         async with self.lock:
             if self.latest_message:
                 msg = self.latest_message
-                logger.debug("Latest message {}: {}", self.platform, msg)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider conditional logging instead of commenting out.

If the intention is to reduce log verbosity, consider implementing a conditional logging mechanism based on a configuration setting or environment variable.

```suggestion
import os

# Assuming the environment variable is named 'DEBUG_LOGGING' and its value is 'true' to enable logging
if os.getenv('DEBUG_LOGGING', 'false').lower() in ['true', '1', 't']:
    logger.debug("Latest message {}: {}", self.platform, msg)
```